sql/lex.h

```diff
@@ -467,6 +467,7 @@ static SYMBOL symbols[] = {
   { "PARTITIONS",       SYM(PARTITIONS_SYM)},
   { "PASSWORD",		SYM(PASSWORD_SYM)},
   { "PERSISTENT",	SYM(PERSISTENT_SYM)},
+  { "PERSISTENT_COUNT", SYM(PERSISTENT_COUNT)},
```

Comment:
I think that this should better be provided as an InnoDB `HA_TOPTION_SYSVAR`, just like `page_compressed`. I believe that MyISAM and Aria always supported a persistent exact count of the rows.

Reply:
I've been working to incorporate persistent count as a `HA_TOPTION_SYSVAR`. Somewhat relating to my 2nd question in the PR's description, setting the `HA_STATS_RECORDS_IS_EXACT` handler flag (as is the case for MyISAM and Aria) for persistent count-enabled tables is janky, since currently persistent count is only relevant for the `READ COMMITTED` mode.

Reply:
can you set or remove `HA_STATS_RECORDS_IS_EXACT` flag depending on the current isolation level?

Reply:
I can try that. Is there any mechanism where a change of isolation level is explicitly registered with `ha_innobase`?